docs/manuals/man1/pki.1.md

```diff
@@ -0,0 +1,350 @@
+pki 1 "February 1, 2019" PKI "pki CLI"
+======================================
```

Comment:
Optional nitpick: you're mixing `=`/`-` style with `#`/`##`/`###` style. It'd be nice to use one style.

Reply:
OK, let me see of the tool accepts that. I just thought the source would look nicer with the main title underlined.

Reply:
Fixed.

Reply:
nit picking: We have 2 of the man pages (`pki-server` and `pki-server-cert`) written in `markdown` format which follows the `=/-`. We can fight over this all day long :stuck_out_tongue: How about we set a standard/template for writing man pages in markdown? :smile:

Reply:
@SilleBille I'm fine with both of the existing man pages because the only have two levels of headings (`=` and `-`), which is the max that format lets you express.

This had three levels of headings... Just to use a consistent style within a page. :)